packages/shared-mocks-lib/src/repo/settlement_batch_repo_mock.ts

```diff
@@ -82,8 +108,15 @@ export class SettlementBatchRepoMock implements ISettlementBatchRepo {
 		toDate: number,
 		model: string,
 		currencyCodes: string[],
-		batchStatuses: string[]
-	): Promise<ISettlementBatch[]> {
+		batchStatuses: string[],
+		pageIndex: number = 0,
+        pageSize: number = MAX_ENTRIES_PER_PAGE,
+	): Promise<BatchSearchResults> {
+		pageIndex = Math.max(pageIndex, 0);
```

Comment:
In these methods, because we call them in many places in the aggragte, we need the default to be return all (without pagination)